plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -192,9 +192,36 @@ function generateAtomDocument() {
 				$pkpDetails->setAttribute('checksumValue', md5_file($packageFile));
 				break;
 		}
-
+		
 		$entry->appendChild($pkpDetails);
 		$atom->appendChild($entry);
+		
+		$license = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'license');
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'openAccessPolicy', $journal->getLocalizedSetting('openAccessPolicy')));
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'licenseURL', $journal->getSetting('licenseURL')));
+		
+		$mode = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'publishingMode');
+		$mode->setAttribute('mode', $journal->getSetting('publishingMode'));
+		switch($journal->getSetting('publishingMode')) {
+			case 0:
+				$mode->nodeValue = 'Open';
+				break;
+			case 1:
+				$mode->nodeValue = 'Subscription';
+				break;
+			case 2:
+				$mode->nodeValue = 'None';
+				break;
+			default:
+				$mode->nodeValue = 'Unknown';
```

Comment:
Well, I think I did. Changed in the source code, committed, rebased, and pushed. But the default clause still shows up above.

SFU is having some network errors, maybe that's the problem?